src/User/User.js

```diff
@@ -12,13 +12,13 @@ class User extends Component {
   render() {
    return (
       <section className="user-view">
-        <article className="user-favs">
+        <article className="user-favs" alt="favorite jokes">
```

Comment:
What is the purpose of these alt tags?  I don't see the need for them on an article